azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/action/ParticipantAction.kt

```diff
@@ -3,4 +3,9 @@
 
 package com.azure.android.communication.ui.chat.redux.action
 
-internal sealed class ParticipantAction : Action
+import com.azure.android.communication.ui.chat.models.RemoteParticipantsInfoModel
+
+internal sealed class ParticipantAction : Action {
+    class ParticipantsAdded(val participants: RemoteParticipantsInfoModel) : ParticipantAction()
```

Comment:
call it infoModel instead of Participants, so later in the code we don't have 
participants.participants.

Reply:
Included in https://github.com/Azure/communication-ui-library-android/pull/481/commits/64ff3df4580e4f182190fdaea4101bdbe4e36307